src/Mailinglist/Client/MailingJetClient.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailinglist\Client;
+
+use Mailjet\Client;
+use Mailjet\Resources;
+
+class MailingJetClient implements MailinglistClient
+{
+    private Client $client;
+
+    public function __construct(string $apiKey, string $apiSecret)
+    {
+        $this->client = new Client(
+            $apiKey,
+            $apiSecret
+        );
+    }
+
+    public function subscribe(string $email, string $mailingListId): void
```

Comment:
Maybe we could add a check to see if `$email` is a valid e-mail, I'm sure Mailjet does it too, but maybe it's best we do not create too much errors on the external server.

Reply:
I think that makes sense, see https://github.com/cultuurnet/udb3-backend/pull/1686#discussion_r1565558363